libr/core/cmd_debug.c

```diff
@@ -2729,17 +2729,17 @@ static int cmd_debug_step (RCore *core, const char *input) {
 	case 'u':
 		switch (input[2]) {
 		case 'f':
-			step_until_flag (core, input+3);
+			step_until_flag (core, input + 3);
 			break;
 		case 'i':
-			step_until_inst (core, input+3);
+			step_until_inst (core, input + 3);
 			break;
 		case 'e':
-			step_until_esil (core, input+3);
+			step_until_esil (core, input + 3);
 			break;
 		case ' ':
 			r_reg_arena_swap (core->dbg->reg, true);
-			step_until (core, r_num_math (core->num, input+2)); // XXX dupped by times
+			step_until (core, r_num_math (core->num, input + 2)); // XXX dupped by times
```

Comment:
yeah just a random comment. np

> On 21 Jun 2016, at 01:48, Joshua J. Drake <notifications@github.com> wrote:
> 
> In libr/core/cmd_debug.c <https://github.com/radare/radare2/pull/5185#discussion_r67787279>:
> 
> >  			break;
> >  		case ' ':
> >  			r_reg_arena_swap (core->dbg->reg, true);
> > -			step_until (core, r_num_math (core->num, input+2)); // XXX dupped by times
> > +			step_until (core, r_num_math (core->num, input + 2)); // XXX dupped by times
> There's not really a great place to declare it here. Anyway I'd be in favor of something larger that made command argument handling much cleaner throughout. I just don't think this PR is the right place for it.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub <https://github.com/radare/radare2/pull/5185/files/e9879b72dc00a8bffb17ddb925ed028cf4920275..0f46d50f5c2d404d6e0546048cdbdbaec5f7987b#r67787279>, or mute the thread <https://github.com/notifications/unsubscribe/AA3-liqsNYHr1d0h64dwhyWARX23KHVBks5qNybRgaJpZM4I6ILO>.
>